tik/tik.cpp

```diff
@@ -336,53 +334,3 @@ int main(int argc, char *argv[])
     }
     return EXIT_SUCCESS;
 }
-
-void CleanModule(Module *M)
-{
-    for (auto mi = M->begin(); mi != M->end(); mi++)
-    {
-        for (auto &fi : *mi)
-        {
-            vector<Instruction *> toRemove;
-            for (auto bi = fi.begin(); bi != fi.end(); bi++)
-            {
-                auto v = cast<Instruction>(bi);
-                if (auto ci = dyn_cast<DbgInfoIntrinsic>(v))
-                {
-                    toRemove.push_back(ci);
-                }
-                else
-                {
-                    SmallVector<std::pair<unsigned, MDNode *>, 1> MDs;
-                    v->getAllMetadata(MDs);
-                    for (auto MD : MDs)
-                    {
-                        v->setMetadata(MD.first, nullptr);
-                    }
-                }
-            }
-            for (auto r : toRemove)
-            {
-                r->eraseFromParent();
-            }
-        }
-        auto *F = cast<Function>(mi);
-        SmallVector<std::pair<unsigned, MDNode *>, 1> MDs;
-        F->getAllMetadata(MDs);
-        for (auto MD : MDs)
-        {
-            F->setMetadata(MD.first, nullptr);
-        }
-    }
-
-    for (auto gi = M->global_begin(); gi != M->global_end(); gi++)
-    {
-        auto gv = cast<GlobalVariable>(gi);
-        SmallVector<std::pair<unsigned, MDNode *>, 1> MDs;
-        gv->getAllMetadata(MDs);
-        for (auto MD : MDs)
-        {
-            gv->setMetadata(MD.first, nullptr);
-        }
-    }
-}
```

Comment:
I know why you needed to remove this, but did you replace it elsewhere? We need to remove the metadata somehow.

Reply:
It's been moved to Annotate. I use it in tikSwap so I thought moving it to AtlasUtil made sense. If you look at AtlasUtil/Annotate.h in Files Changed you'll see it

Reply:
Cool. So where do you call it now? Was that moved?

Reply:
Called in tik.cpp and tikSwap.cpp. Basically used for the same purpose in both cases